src/main/java/com/odysseusinc/arachne/executionengine/api/v1/AnalisysController.java

```diff
@@ -85,12 +85,13 @@ public AnalysisRequestStatusDTO analyze(
             @RequestPart("file") List<MultipartFile> files,
             @RequestHeader(value = "arachne-compressed", defaultValue = "false") Boolean compressed,
             @RequestHeader(value = "arachne-waiting-compressed-result", defaultValue = "false") Boolean waitCompressedResult,
+            @RequestHeader(value = "arachne-datasource-check", defaultValue = "false") Boolean healthCheck,
```

Comment:
attachMetadata

Reply:
Do you expect renaming of "healthCheck" or what do you mean?

Reply:
"attachCdmMetadata"?